crates/api_models/src/payments.rs

```diff
@@ -184,7 +184,12 @@ pub struct PaymentsRequest {
     pub statement_descriptor_suffix: Option<String>,
 
     /// You can specify up to 50 keys, with key names up to 40 characters long and values up to 500 characters long. Metadata is useful for storing additional, structured information on an object.
-    pub metadata: Option<Metadata>,
+    #[serde(flatten)]
+    pub metadata_internal: Option<Metadata>,
```

Comment:
Follow the specs to name the fields and structs.

This field is metadata only.